experimental/traffic-portal/src/app/core/users/tenants/tenants.component.ts

```diff
@@ -80,10 +80,12 @@ export class TenantsComponent implements OnInit, OnDestroy {
 
 	public readonly contextMenuItems: ContextMenuItem<Readonly<Tenant>>[] = [
 		{
+			disabled: (): boolean => !this.auth.hasPermission("TENANT:UPDATE"),
 			href: (t: Tenant): string => `core/tenants/${t.id}`,
 			name: "View Details"
 		},
 		{
+			disabled: (): boolean => !this.auth.hasPermission("TENANT:UPDATE"),
```

Comment:
`disabled` should be used when the action doesn't make sense on the selected data; controls that the user doesn't have Permissions to use should just be hidden.